packages/flutter/lib/src/cupertino/icons.dart

```diff
@@ -99,4 +103,41 @@ class CupertinoIcons {
 
   /// A solid down arrow.
   static const IconData down_arrow = const IconData(0xf35d, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A charging battery.
+  static const IconData battery_charging = const IconData(0xf111, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// An empty battery.
+  static const IconData battery_empty = const IconData(0xf112, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A full battery.
+  static const IconData battery_full = const IconData(0xf113, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A 75% charged battery.
+  static const IconData battery_75_percent = const IconData(0xf114, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A 25% charged battery.
+  static const IconData battery_25_percent = const IconData(0xf115, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// The bluetooth logo.
+  static const IconData bluetooth = const IconData(0xf116, fontFamily: iconFont, fontPackage: iconFontPackage);
+
+  /// A rewind arrow, pointing downwards.
+  static const IconData rewind_arrow = const IconData(0xf21c, fontFamily: iconFont, fontPackage: iconFontPackage);
```

Comment:
I was expecting F4A0 to be the rewind icon. This is probably `restart` or something